### ISSUE TITLE
Suggestion: Rename this repo

### ISSUE THREAD

**Issue by mrchief** [isaacs/github#1559]
The repo name makes it look like it's the official place to raise issues and that it's being monitored by GitHub. The readme busts that myth but people landing on the various issue pages from Google/search engines may not always read the readme.

If possible, a more explicit, conspicuous name would help raise awareness.

If a rename is too disruptive, how about a bot generated message whenever a new issue is opened that basically says: 

> This is not the actual repository for the GitHub website.
> You should also email support@github.com, since this repo is strictly for our own (unofficial) tracking purposes.
> For details, see the [README](link/to/readme)

This brings the awareness closer to the issue page at least.

**Comment by oprogramador** [comment]
@mrchief 

I undertand your concern but IMO `This is not the actual repository for the GitHub website.` is very clear.

**Comment by mrchief** [comment]
@oprogramador Yes, it is. I'm trying to point out the fact that folks may not be reading that. If you click on a google search result such as `https://github.com/isaacs/github/issues/900` - then you go straight to the issue page. All you see there is the issue and the repo name on the top like so

![image](https://user-images.githubusercontent.com/781818/57053846-b15e8c80-6c5e-11e9-9d6f-5eb0e548a7f2.png)

One may think (wrongly) that GitHub is monitoring this and that upvoting here will push the issue up the priority list on GitHub's side. However, that is not the case and that is not immediately obvious (unless one bothers to visit the README). 

I'm not saying everyone misses that. It can happen unintentionally or because people are busy, they may not pay much attention. A little nudge towards the truth will only prompt more folks to contact GitHub's support the official way and may help in a speedier resolution of issues. 

There is no absolute way to quantify or measure the impact here either way, hence it's a suggestion. Renaming might be a bit too much and I realize that, which is why I also suggested using something like [Welcome Bot](https://probot.github.io/apps/welcome/) to educate/remind folks about contacting real GitHub support as well. 

If you think the status quo is sufficient, then you can close this.

**Comment by TPS** [comment]
A practical concern: IMHO, sadly, [this is 1 of the most-linked GitHub repos](https://www.google.com/search?q=link%3Agithub.com%2Fisaacs%2Fgithub+-site%3Agithub.com%2Fisaacs%2Fgithub).  Changing the repo name would likely break ½ _million_+ links (even excluding linking from _w/in_ the repo itself).

**Comment by oprogramador** [comment]
@TPS 

Renaming should work because GitHub redirects from the old name to the new name.
